WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
+        // If a capability is missing it means the user don't have it.
+        if (capabilities == null || capabilities.isEmpty() || capability == null) {
+            return false;
+        }
+        try {
+            JSONObject jsonObject = new JSONObject(capabilities);
```

Comment:
I was mostly referring to the "rest" of the capabilities (like `edit_users` and others that are very likely we will employ soon) but I agree that all in all we're probably gonna be loading the model more often than we'll need the capabilities checked.

I did consider the option of caching the object on first use but opted against suggesting it because of the ugliness of the code around how to determine whether the object is cached or not (checking for null won't be enough since the json can be malformed, leading to null anyway). It would need a boolean instance variable to flag the fact we tried to load the json and that was the bit I considered ugly and dropped the suggestion. That was a thought experiment though so I could be wrong and this can end up be nice!